.babelrc

```diff
@@ -3,11 +3,12 @@
     "react"
   ],
   "plugins": [
-    "transform-es2015-modules-amd",
     "check-es2015-constants",
     "transform-es2015-block-scoping",
     "transform-es2015-arrow-functions",
     "transform-es2015-parameters",
+    "transform-es2015-destructuring",
```

Comment:
Nope :/ Just tried, it requires few more things (at least I've noticed that Edge doesn't support "includes")